addons/event/models/event_mail.py

```diff
@@ -118,13 +162,28 @@ def execute(self):
                     mail.write({'mail_registration_ids': lines})
                 # execute scheduler on registrations
                 mail.mail_registration_ids.filtered(lambda reg: reg.scheduled_date and reg.scheduled_date <= now).execute()
+
+                contacted_attendees = len(mail.mail_registration_ids.filtered(lambda reg: reg.mail_sent))
+                mail.contact_count_str = attendees_str % contacted_attendees if contacted_attendees else False
             else:
                 # Do not send emails if the mailing was scheduled before the event but the event is over
                 if not mail.mail_sent and (mail.interval_type != 'before_event' or mail.event_id.date_end > now) and mail.notification_type == 'mail':
                     mail.event_id.mail_attendees(mail.template_id.id)
                     mail.write({'mail_sent': True})
+
+                    contacted_attendees = len(mail.event_id.registration_ids.filtered(lambda reg: reg.state != 'cancel'))
+                    mail.contact_count_str = attendees_str % contacted_attendees if contacted_attendees else False
+
         return True
 
+    def _get_attendees_str(self):
```

Comment:
If we store an integer we could get rid of that strange code ... better let interface handle interface.

Reply:
@tde-banana-odoo But so we will need either a computed field or a widget... Maybe a computed field based on the language of the current user is better ?

Reply:
Can't we simply have "# Contacted Attendees" as column header and simply have a count in rows ? Seems easier than storing a sentence.